ext/nnstreamer/tensor_filter/tensor_filter_trix_engine.hh

```diff
@@ -0,0 +1,78 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2020 Samsung Electronics
+ * Copyright (C) 2020 Dongju Chae <dongju.chae@samsung.com>
+ */
+/**
+ * @file    tensor_filter_subplugin_trix_engine.hh
+ * @date    20 Jan 2020
+ * @brief   NNStreamer tensor-filter subplugin trix_engine header
+ * @see     https://github.com/nnstreamer/nnstreamer
+ * @author  Dongju Chae <dongju.chae@samsung.com>
+ * @bug     No known bugs
+ */
+
+#ifndef __TENSOR_FILTER_SUBPLUGIN_TRIxEngine_H__
+
```

Comment:
plz add 
`#define __TENSOR_FILTER_SUBPLUGIN_TRIxEngine_H__`

and update line 78
`#endif /* __TENSOR_FILTER_SUBPLUGIN_TRIxEngine_H__ */`

Reply:
My bad :) Thanks.